scout/demo/resources/genemap2_reduced.txt

```diff
@@ -1,7 +1,7 @@
 # Copyright (c) 1966-2018 Johns Hopkins University. Use of this file adheres to the terms specified at https://omim.org/help/agreement.
 # Generated: 2018-04-24
 # See end of file for additional documentation on specific fields
-# Chromosome	Genomic Position Start	Genomic Position End	Cyto Location	Computed Cyto Location	Mim Number	Gene Symbols	Gene Name	Approved Symbol	Entrez Gene ID	Ensembl Gene ID	Comments	Phenotypes	Mouse Gene Symbol/ID
+# Chromosome	Genomic Position Start	Genomic Position End	Cyto Location	Computed Cyto Location	MIM Number	Gene Symbols	Gene Name	Approved Symbol	Entrez Gene ID	Ensembl Gene ID	Comments	Phenotypes	Mouse Gene Symbol/ID
```

Comment:
✅

Reply:
Thanks!

Reply:
You did test it, right? 😊

Reply:
I did! If you want to give it a try I can give you @moonso's key! :D